Packs/CommonScripts/Scripts/ParseCSV/ParseCSV.py

```diff
@@ -208,13 +208,17 @@ def main():
                     is_valid = is_ip_valid(row[parse_ip])
                     if is_ip and is_valid:
                         ip_list.append(row[parse_ip])
+                        md += '\n'
+                        continue
 
                 if parse_domain != -1:
                     md += (row[parse_domain] + '|' if row[parse_domain] else ' |')
                     has_dot = '.' in row[parse_domain]
                     no_spaces = ' ' not in row[parse_domain]
                     if has_dot and no_spaces:
                         domain_list.append(row[parse_domain])
+                        md += '\n'
+                        continue
 
                 if parse_hash != -1:
```

Comment:
let's move this if block to be above the parse_domain if block, and handle it similarly

Reply:
done